README.md

```diff
@@ -138,63 +131,163 @@ In support of the various browser privacy models, first-party sets only control
+service</td>
+      <td><br>
+Reserved for utility or sandbox domains.<br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA-usercontent.com<br>
+<br>
+exampleA-cdn.net</td>
+    </tr>
+    <tr>
+      <td><br>
+associated</td>
+      <td><br>
+Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td><br>
+exampleA-affiliated.com<br>
+<br>
+exampleB.com*<br>
+<br>
+exampleC.com*<br>
+<br>
+<br>
+*where exampleB and exampleC are separately owned websites, but clearly present their affiliation with exampleA to users</td>
+    </tr>
+  </tbody>
+</table>
+
+While we think this subset framework has the clear benefit of furthering transparency around why a domain has been added to a set, the primary value to this framework is that the browser could handle each subset differently:
+
+<table>
+  <thead>
+    <tr>
+      <th><strong>Subset type</strong></th>
+      <th><strong>Subset definition</strong></th>
+      <th><strong>Example browser handling policy</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>ccTLD (country code Top Level Domain)</td>
+      <td>Reserved for variations for a particular country or a geographical area. <br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>common eTLD (effective Top Level Domain)</td>
+      <td>Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>service</td>
+      <td>Reserved for utility or sandbox domains.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access. Not allowed to be the top-level domain in a storage access grant.</td>
+    </tr>
+    <tr>
+      <td>associated</td>
+      <td>Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td>Limit of 3* domains. If greater than 3, auto-reject access.<br>
+<br>
+<em>*[^1]exact number TBD</em></a></td>
+    </tr>
+  </tbody>
+</table>
+
+### Abuse mitigation measures
+
+We consider using a public submission process (like a GitHub repository) to be a valuable approach because it facilitates our goal to keep all set submissions public and submitters accountable to users, civil society, and the broader web ecosystem. For example, a mechanism to report potentially invalid sets may be provisioned. We expect public accountability to be a significant deterrent for intentionally miscategorized subsets.
```

Comment:
Public accountability will be valuable as long as the number of sets requiring review is manageable and the review work is seen as meaningful. It would be useful to limit the number of proposed sets requiring human attention, especially for the "associated" type, by requiring associated research and through setting sensible waiting periods for re-submitting the same domain in a new set after a rejection of a previous one.

It is always important to consider the incentives for participating in public accountability. People will be less likely to participate if their efforts are seen as duplicating a task that could have been automated, or as joining an open-ended argument about whether a set is valid. There is likely to be more and higher-quality participation if it is presented as evaluating the existing user research about a set than if it is just an opportunity to express opinions.

There will also need to be appropriately designed and enforced anti-abuse and anti-harassment measures for public review participants. (nobody wants to get SWATted because their comment kept some scammer's bogus set from getting in)

Reply:
I think this is an important point to make, besides your point about user research evaluation, we should think about how to ensure that all humans involved in the public process are appropriately protected and feel like their time is spent well. I can file an issue to that effect!

Reply:
Filed https://github.com/WICG/first-party-sets/issues/105